packages/grafana-ui/src/components/uPlot/config/UPlotSeriesBuilder.ts

```diff
@@ -215,7 +215,7 @@ function mapDrawStyleToPathBuilder(
   barWidthFactor = 0.6,
   barMaxWidth = Infinity
 ): Series.PathBuilder {
-  const pathBuilders = uPlot.paths;
+  const pathBuilders = (require('uplot').default as typeof uPlot).paths;
```

Comment:
Is there a missing change here - uplot is still imported at the top of the file "normally"?

Reply:
Actually, this is not much of a problem: in this case `uPlot` is being imported to be used as a type (the only direct reference to it is `typeof uPlot` so this import won't be taken into account in the bundle